compiler/rustc_middle/Cargo.toml

```diff
@@ -40,5 +40,8 @@ smallvec = { version = "1.8.1", features = ["union", "may_dangle"] }
 thin-vec = "0.2.12"
 tracing = "0.1"
 
+# Not used directly, but included to enable the unstable_offset_of feature
+memoffset = { version = "0.9", features = ["unstable_offset_of"] }
+
```

Comment:
Can this be moved to `rustc_query_impl` where `field-offset` and `memoffset` used?